public/javascripts/impl.validation.js

```diff
@@ -196,8 +196,19 @@
             {
                 return _.all(parentFLs, function(fl) { return fl !== true; });
             },
-            severity: 'warning',
+            warning: true,
```

Comment:
I like the new message. We should use something similar for the error.

**warning**
> Because this control is within a single-screen group (field list), the follow-up question feature will not work (at least in ODK Collect).

**error**
> Because this control is within a single-screen group (field list), any expressions that reference other fields in the same group will not work as expected (at least in ODK Collect).